src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -80,19 +82,28 @@ const ImagesTable = () => {
   const blueprintVersionFilter = useAppSelector(selectBlueprintVersionFilter);
   const blueprintsOffset = useAppSelector(selectOffset);
   const blueprintsLimit = useAppSelector(selectLimit);
-
-  const { selectedBlueprintVersion } = useGetBlueprintsQuery(
+  const { selectedBlueprintVersion } = (
+    isOnPremise === 'cockpit'
+      ? useGetBlueprintsOnPremQuery
+      : useGetBlueprintsQuery
```

Comment:
We really want the hooks to have identical names so that we don't have to have conditionals like this in the code where we differentiate between IB service and IB on-prem.

I think we should look into conditional importing. Then we can import the hook from the service store if it is the service, and import the hook from the on-prem store if it is on-prem.

Reply:
I think this should be possible:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#importing_different_modules_based_on_environment